vfcompositor/Compositor.c

```
@@ -412,16 +439,51 @@ void VFCompositor(void) {
+                if (c == PS2_CalcCombo(K_CTRL, 0x1B)) {
+                    Unsnooze();
+                    ClearScreen();
+                    PrintKernelWarning("VFCompositor: exiting...\n");
+                    VFCompositorShutdown();
+                    KillProcess(GetCurrentProcess()->pid);
+                    return;
+                }
+                if (c == PS2_CalcCombo(K_CTRL, 'N')) {
+                    Window* w = CreateWindow(50, 50, 480, 360, "New Window");
+                    if (w) { w->minimized = false; WindowFill(w, WINDOW_BG); WindowDrawRect(w, 0, 0, w->rect.width, 20, TITLE_BAR); g_focused_window = w; }
+                } else if  (c == PS2_CalcCombo(K_CTRL, 'W')) {
```

Clang (14.0.6)

[note] +1, nesting level increased to 4

(clang)